polliwog/polyline/_polyline_object.py

```diff
@@ -301,6 +301,16 @@ def aligned_with(self, vector):
         else:
             return self
 
+    def aligned_using_points(self, p1: np.ndarray, p2: np.ndarray):
+        """
+        Return a new Polyline which is conditionally flipped so that p1 is
+        sequentially before p2.
+        """
+        return self.flipped_if(
```

Comment:
I think this approach returns something reasonable for open polylines but not closed, since the index doesn't really matter and it's more the distance we'd need to consider